src/syscalls.h

```diff
@@ -51,6 +52,8 @@ struct arch_syscall_table {
 	int aarch64;
 	enum scmp_kver aarch64_kver;
 
+	int loongarch64;
+	enum scmp_kver loongarch64_kver;
```

Comment:
It would be nice to have a blank line after the loongarch64 fields so it is separated from the other ABI/arch groups like the rest of the `arch_syscall_table` struct definition.

Reply:
Fixed, thanks.